client/src/shared/utils/navBarList.ts

```diff
@@ -1,10 +1,16 @@
 export const navItems = [
   { url: '/dashboard', text: 'Dashboard' },
-  { url: '/courses', text: 'Courses' },
+  { url: '/trainer/courses/1/content', text: 'Courses' },

```

Comment:
since your route is not gonna be registered in this file, please remove this changes, we can visit the page by url for now, and link it after courses page's merged.